tests/Runner/TestSuiteSorterTest.php

```diff
@@ -101,10 +101,15 @@ public function testSuiteSorterDefectsOptions(int $order, bool $resolveDependenc
     {
         $suite = new TestSuite;
         $suite->addTestSuite(\MultiDependencyTest::class);
+
         $cache = new TestResultCache();
         $cache->loadFromListenerRunstate($runState);
+
         $sorter = new TestSuiteSorter();
-        $sorter->setCache($cache);
+        $_sorter = new \ReflectionClass($sorter);
```

Comment:
_? leading _?

what about `$sorterReflection` ?

also, having a need to access class' internals in test indicate bad class structure. maybe Cache shall be injected to Sorter via constructor ?